pkg/api/v1/BUILD

```diff
@@ -72,7 +72,7 @@ filegroup(
         ":package-srcs",
         "//pkg/api/v1/endpoints:all-srcs",
         "//pkg/api/v1/helper:all-srcs",
-        "//pkg/api/v1/node:all-srcs",
+        "//pkg/util/node:all-srcs",
```

Comment:
This file is auto-generated. Please don't manually modify it. Run `./hack/update-bazel.sh` to update.

Also please split these auto-generated files into a separate commit.